jupyter-notebooks/analytics/5. Summary Statistics Ships and Clouds.ipynb

```diff
@@ -0,0 +1,694 @@
+    "\n",
+    "The workflow involves:\n",
+    "- Posting a stats job request\n",
+    "- Polling the job stats endpoint\n",
+    "- Getting the job report results\n",
+    "- Restructuring the results into a pandas dataframe\n",
+    "- Visualizing the time series"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Post a stats job request"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### a) Define your subscription"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ships_subscription_id='d493559c-2d9b-47e7-bd35-c04351544102'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### b) Visualize Subscription Bounds"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 39,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import requests\n",
+    "import json\n",
+    "import pprint\n",
+    "pp = pprint.PrettyPrinter(indent=4)\n",
+    "\n",
+    "def jpp(data):\n",
+    "    print(json.dumps(data, indent=4))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "User Guide: https://github.com/keplergl/kepler.gl/blob/master/docs/keplergl-jupyter/user-guide.md\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "45e39212da9e4d41b101801c25d63973",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "KeplerGl(data={'Bandar Abbas Subscription d493559c-2d9b-47e7-bd35-c04351544102': {'index': [0], 'columns': ['g…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Get Subscription Geometry from Feeds API\n",
+    "API_KEY = os.environ['PL_API_KEY']\n",
+    "BASIC_AUTH = (API_KEY, '')\n",
+    "session = requests.Session()\n",
+    "session.auth=BASIC_AUTH\n",
+    "\n",
+    "SIF_BASE_URL = 'https://sif-live.prod.planet-labs.com/'\n",
+    "subscription_get_url = SIF_BASE_URL + f'subscriptions/{ships_subscription_id}'\n",
+    "\n",
+    "subs_get_resp = requests.get(\n",
+    "    subscription_get_url, \n",
+    "    auth=BASIC_AUTH\n",
+    ")\n",
+    "\n",
+    "# Convert geometry to a Geopandas DataFrame\n",
+    "subscription_geometry = subs_get_resp.json()['geometry']\n",
+    "subscription_gdf = gpd.GeoDataFrame.from_file(json.dumps(subs_get_resp.json()))\n",
+    "\n",
+    "# Visualize Geometry with KeplerGL\n",
+    "subscription_map = KeplerGl(height=500)\n",
```

Comment:
KeplerGl is not imported in this notebook
also it does not appear to be installed in the planet notebooks docker image `planetlabs/notebooks`